definitions/infra-awsrekognition/golden_metrics.yml

```diff
@@ -0,0 +1,36 @@
+requests:
+  title: Total Request Count
```

Comment:
For consistency, the word "Count" should be lowercase

Reply:
addressed